.github/workflows/linux_ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    name: Ubuntu Tests
+    env:
+      ENV_NAME: hera_qm_tests
+      PYTHON: ${{ matrix.python-version }}
+      OS: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
if you're feeling feisty you can enable macos-latest as an OS, but you only get 5 concurrent mac builds at a time. and well mac can be testy some times.

Reply:
Let's give it a try!
I don't think many people are running qm on macs (mostly on nrao/enterprise/on site), but might as well keep track of it. We can always run it but not make it required.